draft-ietf-quic-recovery.md

```diff
@@ -661,19 +662,27 @@ receives packets with the Congestion Experienced codepoint.
 
 ## Slow Start
 
-QUIC begins every connection in slow start and exits slow start upon loss or
-upon increase in the ECN-CE counter. QUIC re-enters slow start any time the
-congestion window is less than ssthresh, which only occurs after persistent
-congestion is declared. While in slow start, QUIC increases the congestion
-window by the number of bytes acknowledged when each acknowledgment is
-processed.
+QUIC begins every connection in slow start with the congestion window set to
+an initial value.  The RECOMMENDED initial value is the minimum of
+10 * max_datagram_size and max(2 * max_datagram_size, 14720)).  This follows
+the analysis and recommendations in {{?RFC6928}}, increasing the byte limit to
```

Comment:
The justification is repeated in Appendix B.1. I like it better here, but maybe make B.1 more concise and just point to here?

Reply:
SG, I made B.1 more consise and gave this and the min CWND their own sub-section.